django/test/runner.py

```diff
@@ -336,12 +336,20 @@ def run(self, test):
         return result
 
 
-def parallel_type(value):
-    """Parse value passed to the --parallel option."""
+def get_num_test_processes(value=None):
+    """Number of test processes when using the --parallel option."""
```

Comment:
Maybe document that the value passed to this function should be a value parsed from the `--parallel` option? You can also mention the max info I mentioned above.